sitemap.md

```diff
@@ -1,8 +1,10 @@
  * [Home](index.md)
  * [User's Guide](users-guide.md)
+   * [Dagger & Android](android.md)
```

Comment:
That's true, but it was never added to the sync tarball.  That can happen later.  It's irrelevant to this bit, though, insofar as this sitemap.md isn't used in the gh-pages jekyll template.